Zewa/Database.php

```diff
@@ -36,12 +36,12 @@ public function establishConnection($name = 'default')
                     self::$dbh[$name] = new \PDO($dbConfig->dsn, $dbConfig->user, $dbConfig->pass);
                     self::$dbh[$name]->setAttribute(\PDO::ATTR_ERRMODE, \PDO::ERRMODE_EXCEPTION);
                 } else {
-                    throw new Exception\TypeException('Please specify a valid database configuration, or provide a default configuration.');
+                    throw new \PDOException('Please specify a valid database configuration, or provide a default configuration.');
                 }
 
             }
 
-        } catch (Exception\TypeException $e) {
+        } catch (\PDOException $e) {
```

Comment:
You don't create exceptions just to catch them in the next line of code.  This doesn't make sense.  You've added a ton of unnecessary code with these try/catch loops.  Remove them entirely.